packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -59,6 +59,11 @@ RocketChat.statistics.get = function _getStatistics() {
 		uptime: process.uptime()
 	};
 
+	statistics.deploy = {
+		method: process.env.DEPLOY_METHOD || 'tar',
+		platform: process.env.DEPLOY_PLATFORM || 'selfinstall'
```

Comment:
should we check for valid methods and platforms instead of assuming that any value is valid?